src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -15,18 +12,18 @@ trait Attributable extends Identifiable {
  */
 case class WorkspaceName(
                       namespace: String,
-                      name: String) extends Identifiable {
-  def path : String = "workspaces/" + namespace + "/" + name
+                      name: String) {
   override def toString = namespace + "/" + name // used in error messages
+  def path = s"/workspaces/${namespace}/${name}"
 }
 
 case class WorkspaceRequest (
                       namespace: String,
                       name: String,
                       attributes: Map[String, Attribute]
-                      ) extends Identifiable with Attributable {
-  def path = WorkspaceName(namespace,name).path
+                      ) extends Attributable {
   def toWorkspaceName = WorkspaceName(namespace,name)
```

Comment:
i guess, technically we do have those two things as namespace, name, if we had a function that returned a tuple of them would you call that workspace.toTupleOfNamespaceAndName?